### ISSUE TITLE
XEPs and RFCs support page with version

### ISSUE THREAD

**Issue by Neustradamus** [xmppjs/xmpp.js#531]
It is possible to have a XEPs and RFCs support page with version?

Examples:
- https://metronome.im/supported-xeps-list
- https://trac.gajim.org/wiki/GajimXEPSupport
- https://github.com/jappix/jappix/blob/master/PROTOCOL.md

**Comment by mickeprag** [comment]
I think this is a must.

When/if someone starts working on such page then we have started som work on a couple XEPs:
https://github.com/telldus/xmppjs-client-plugins

**Comment by Neustradamus** [comment]
@mickeprag: Whaouuuuuuuuuu! <3
Have you seen with xmppjs team (@astro, @chris-rock, @dodo, @ggozad, @hyperlink, @lloydwatkin, @sonnyp) to integrate?

If you are ready to add a real SCRAM-SHA support: https://github.com/xmppjs/xmpp.js/issues/620

**Comment by ggozad** [comment]
Yeah we had suggested to do that before as well under the xmppjs org. 
Here's some we did: https://github.com/Crypho/xmpp-plugins

**Comment by Neustradamus** [comment]
Nice @ggozad, hope good changes from @sonnyp soon :)

**Comment by sonnyp** [comment]
Of course, I'm open to have a list of community plugins/XEPs to README.md
Feel free to send a PR.

Regarding hosting/maintaining them under the xmpp.js organization unfortunately I don't think the project is quite ready. There are a few things to figure out such as APIs/conventions/level of abstraction, TypeScript?, documentation, modules, ...

I'm quite busy ATM but I'll try move things forward by the end of the year.

@mickeprag cool stuff :+1:

**Comment by sonnyp** [comment]
WIP https://github.com/xmppjs/xmpp.js/blob/master/protocols.md

Feel free to PR (third party should be in a separate section)

**Comment by FlorianLudwig** [comment]
@sonnyp is it still "WIP" or is completed and reflects the current state of xmpp.js? If so it would be great to link it directly in the readme :)

Thanks!

**Comment by Neustradamus** [comment]
@sonnyp: Thanks but versions of XEPs are missing :/
Can you add it?

Do not forget to plan the .doap file to be listed in xmpp.org like others, example:
- https://xmpp.org/extensions/#xep-0384-implementations
